SSPSolution/ResourceLib/FileLoader.cpp

```diff
@@ -167,13 +165,64 @@ Resources::Status Resources::FileLoader::LoadFile(std::string & path, char *& da
 	return Resources::Status::ST_OK;
 }
 
+DLL_OPERATION Resources::Status Resources::FileLoader::LoadLevel(std::string & path, LevelData::Level *& levelPtr)
```

Comment:
DLL_OPERATION is already called in the header for this function and headers are the only place where we should use this macro. Please correct this.
Just a side note, wouldn't this prompt a warning for calling it twice?